### PR TITLE
Code view#806 Edit Example: Important Words

### PR DESCRIPTION
**Pull request by a13alfbe**
This is only one part of issue #806. I've added functionality to add and remove important words for code examples. 

Don't close the issue after the review is done. I will continue working on this issue

### CODE REVIEW

File: CodeViewer/codeviewer.js [hunk 1]
```diff
@@ -197,6 +197,40 @@ function returned(data)
...
+			$("#impwords").append('<option>' + word + '</option>');
+			$("#impword").val("");
+			addedWords.push(word);
+		}
+	}
+
+	else if (editType == "-") {
+		word = $('option:selected', "#impwords").text();
+		$('option:selected', "#impwords").remove();
+    	removedWords.push(word);
```
Comment (b12simsa) [inline CodeViewer/codeviewer.js]: Line 228 might be missing a tab.

File: Shared/dugga.js [hunk 2]
```diff
@@ -183,13 +183,22 @@ function htmlEntities(str) {
...
-function AJAXService(opt,apara,kind)
+function AJAXService(opt,apara,kind)	
 {
 	var para="";
 	for (var key in apara) {
-			para+="&"+key+"="+encodeURIComponent(htmlEntities(apara[key]));
-	}
-				
+			// Handles all the individual elements in an array and adds the array as such: &key=val1,val2,val3
+			if (apara[key].constructor === Array) {
```
Comment (b12simsa) [inline Shared/dugga.js]: If a user opens the EditorV50.php, an error message is displayed. The message probably has something to do with this line.
errordump2

**Comment by b12simsa** [comment]
errordump3

The CodeViewer displayed some unexpected behavior, see the above image.
The following important words were entered in the listed order.

1. "start"
2. "startY"
3. "1"

The save button were then repetedly pressed until the browser crashed.

**Comment by a13alfbe** [comment]
Problems are now fixed (i hope)!

**Comment by a13alfbe** [comment]
Fixed a bug with deleting words

**Comment by a13alfbe** [comment]
Added support for converting "/", "\" and "?" to HTML entities.

**Comment by b12simsa** [comment]
After the changes the page is stable, I cannot find any symbols that causes crashes. Think this can be merged with the week branch now.

**Comment by a13henhu** [comment]
I have reviewed this, and it is functional correctly, and just awaiting for merging conformation

**Pull request merged**